packages/cozy-harvest-lib/src/components/KonnectorConfiguration/ConfigurationTab/index.jsx

```diff
@@ -40,61 +44,63 @@ const ConfigurationTab = ({
   const hasLoginError = error && error.isLoginError()
 
   return (
-    <div className="u-stack-m">
-      {shouldDisplayError && hasLoginError && (
-        <TriggerErrorInfo
-          className="u-mb-2"
-          error={error}
-          konnector={konnector}
-        />
-      )}
-      {!konnector.oauth ? (
-        <div>
-          <ListSubHeader>{t('modal.updateAccount.title')}</ListSubHeader>
-          <List dense className="u-pt-0">
-            <ListItem
-              className="u-mt-half u-c-pointer"
-              onClick={() => pushHistory(`/accounts/${account._id}/edit`)}
-            >
-              <ListItemIcon>
-                <Icon icon="lock" color={palette['coolGrey']} />
-              </ListItemIcon>
-              <ListItemText
-                primaryText={konnector.name}
-                secondaryText={Account.getAccountName(account)}
-              />
-              <ListItemSecondaryAction>
-                {running && <Spinner />}
-              </ListItemSecondaryAction>
-              <ListItemSecondaryAction>
-                <Icon
-                  className="u-mr-1"
-                  icon="right"
-                  color={palette['coolGrey']}
+    <ModalContent className={'u-p-0'}>
```

Comment:
{''} or "" ? You seems to mix things up on this point?

Reply:
Do you mean that I should write `className='u-p-0'` ? Fixed.